table.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE Account(
+    username VARCHAR(256) NOT NULL PRIMARY KEY,
+    email VARCHAR(64) NOT NULL UNIQUE,
+    password VARCHAR(256) NOT NULL
+);
+
+CREATE TABLE Game(
+    game_id INTEGER NOT NULL PRIMARY KEY,
+    score INTEGER,
+    mode VARCHAR(20) NOT NULL
+);
+
+CREATE TABLE Records(
+    username VARCHAR(256) NOT NULL PRIMARY KEY REFERENCES Account(username),
+    high_score INTEGER,
+    avg_score INTEGER,
+    games_played INTEGER
+);
+
+CREATE TABLE Location(
+    image_id INTEGER NOT NULL PRIMARY KEY,
+    image_file VARCHAR(256) NOT NULL UNIQUE,
+    lat DECIMAL(8,6) NOT NULL UNIQUE,
+    long DECIMAL(9,6) NOT NULL UNIQUE,
+    game_mode VARCHAR(20)
```

Comment:
I think we might want longer game mode here and we can store all the game modes that it's valid for inside it.
E.g., "bathroom,inside,west" 

We also could store a table of game mode per image? And then j have one entry for each mode that's valid